lib/wizard_timeline_manager.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
For the other classes that exceed the rubocop length, I've done the disabling in `.rubocop.yml` instead, so that there's a central place to see all the over-length classes.

Reply:
Ah nice! I will check that out and change. 👍